src/Conf.ml

```diff
@@ -189,8 +189,14 @@ end = struct
   let choice ~all ~names ~doc ~section
       ?(allow_inline = Poly.(section = `Formatting)) update =
     let open Cmdliner in
-    let _, default, _ = List.hd_exn all in
-    let doc = generated_choice_doc ~allow_inline ~all ~doc ~section in
+    let default_name, default, _ = List.hd_exn all in
+    let opt_default, all =
+      if String.equal default_name "IGNORE" then (None, List.tl_exn all)
```

Comment:
Have you considered adding an optional boolean arg to `choice` that could be passed in the profile case and branched on here?